modules/p2-profile-gen/pom.xml

```diff
@@ -182,6 +182,9 @@
                                 <featureArtifactDef>
                                     org.wso2.carbon.identity.event.handler.notification:org.wso2.carbon.event.handler.notification.server.feature:${identity.event.handler.notification.version}
                                 </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity.framework:org.wso2.carbon.identity.user.action.feature:${carbon.identity.framework.version}
+                                </featureArtifactDef>
```

Comment:
Isn't this added with core feature @ashanthamara

Reply:
Yeah will get these changes merged separately.
https://github.com/wso2/product-is/pull/22242